jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -47,6 +47,7 @@ def inside_git_repo(self, current_path):
     try:
```

Comment:
The doc string should be one sentence. You can put additional description in a separate paragraph.

```
def f():
  ""one sentence doc string.

  More description here in a separate paragragh.
  ""
```